qt_gui/src/qt_gui/plugin_handler.py

```diff
@@ -109,7 +109,10 @@ def _emit_load_completed(self, exception=None):
         if self.__callback is not None:
             callback = self.__callback
             self.__callback = None
-            callback(self, exception)
+            try:
+                callback(self, exception)
+            except TypeError:
```

Comment:
Can you please provide more information in which cases the callback can raise a `TypeError` and why it should be handled differently from other exceptions.